### PR TITLE
MULE-8392: Update developers list in Mule pom

### PR DESCRIPTION
**Pull request by estebanwasinger**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -79,6 +79,18 @@
             <id>mbuchwald</id>
             <name>Martin Buchwald</name>
         </developer>
+        <developer>
+            <id>alegmarra</id>
+            <name>Alejandro Garcia Marra</name>
+        </developer>
+        <developer>
+            <id>estebanwasinger</id>
```
Comment (fernandezlautaro) [inline pom.xml]: isn't this one responsible of instagram's account rather than developing in mule?

Comment (estebanwasinger) [reply pom.xml]: both things 📸

Comment (estebanwasinger) [reply pom.xml]: follow @mulesoftarg

Comment (juandesi) [reply pom.xml]: Garcia Marra -> Gamarra

Comment (fernandezlautaro) [reply pom.xml]: ☝️ , `Gamarra` FTW.

**Pull request merged**